wfd/common/message_handler.h

```diff
@@ -79,6 +78,14 @@ class MessageHandler {
     assert(observer_);
   }
 
+  inline SourceMediaManager* GetSourceMediaManager() {
```

Comment:
make it a free function (located in the same place where SourceMediaManager is declared)